ui/responsive/app/components/account-dropdowns.js

```diff
@@ -0,0 +1,217 @@
+const Component = require('react').Component
+const PropTypes = require('react').PropTypes
+const h = require('react-hyperscript')
+const actions = require('../actions')
+const genAccountLink = require('../../lib/account-link.js')
+const connect = require('react-redux').connect
+const Dropdown = require('./dropdown').Dropdown
+const DropdownMenuItem = require('./dropdown').DropdownMenuItem
+const Identicon = require('./identicon')
+const ethUtil = require('ethereumjs-util')
+const copyToClipboard = require('copy-to-clipboard')
+
+class AccountDropdowns extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      overflowMenuActive: false,
+      switchingMenuActive: false,
+    }
+  }
+
+  getAccounts () {
+    const { identities, selected } = this.props
+
+    return Object.keys(identities).map((key) => {
+      const identity = identities[key]
+      const isSelected = identity.address === selected
+
+      return h(
+        DropdownMenuItem,
+        {
+          closeMenu: () => {},
+          onClick: () => {
+            this.props.actions.showAccountDetail(identity.address)
+          },
+        },
+        [
+          h(
+            Identicon,
+            {
+              address: identity.address,
+              diameter: 16,
+            },
+          ),
+          h('span', { style: { marginLeft: '10px' } }, identity.name || ''),
+          h('span', { style: { marginLeft: '10px' } }, isSelected ? h('.check', '✓') : null),
```

Comment:
Went for a helpful touch here, in case the user gets confused with similar account names. We pass `selected` from the container and render a checkmark for the selected account. 👍  👎 ?
![](https://puu.sh/wMF3c/504b1e48c6.png)